app/experimenter/kinto/serializers.py

```diff
@@ -0,0 +1,75 @@
+from time import time
+from rest_framework import serializers
+
+from experimenter.experiments.models import Experiment, ExperimentVariant
+
+
+class ExperimentRapidBranchesSeraizlier(serializers.ModelSerializer):
+    value = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ExperimentVariant
+        fields = ("slug", "ratio", "value")
+
+    def get_value(self, obj):
+        # placeholder value
+        return {}
+
+
+class ExperimentRapidArgumentSerializer(serializers.ModelSerializer):
+    slug = serializers.ReadOnlyField(source="normandy_slug")
+    userFacingName = serializers.ReadOnlyField(source="name")
+    userFacingDescription = serializers.ReadOnlyField(source="public_description")
+    active = serializers.ReadOnlyField(default=True)
+    isEnrollmentPaused = serializers.ReadOnlyField(default=False)
+    proposedEnrollment = serializers.ReadOnlyField(source="proposed_enrollment")
+    bucketConfig = serializers.SerializerMethodField()
+    startDate = serializers.SerializerMethodField()
+    endDate = serializers.ReadOnlyField(default=None)
+    branches = ExperimentRapidBranchesSeraizlier(many=True, source="variants")
+    referenceBranch = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Experiment
+        fields = (
+            "slug",
+            "userFacingName",
+            "userFacingDescription",
+            "active",
+            "isEnrollmentPaused",
+            "features",
+            "proposedEnrollment",
+            "bucketConfig",
+            "startDate",
+            "endDate",
+            "branches",
+            "referenceBranch",
+        )
+
+    def get_bucketConfig(self, obj):
+        return {
+            "randomizationUnit": "normandy_id",
+            "namespace": "",
+            "start": 0,
+            "count": 0,
+            "total": 10000,
+        }
+
+    def get_referenceBranch(self, obj):
+        control_branch = obj.variants.get(is_control=True)
+        return control_branch.slug
+
+    def get_startDate(self, obj):
+        # placeholder value
+        return int(time())
+
+
+class ExperimentRapidRecipeSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField(source="normandy_slug")
+    arguments = ExperimentRapidArgumentSerializer(source="*")
```

Comment:
What does the `*` do?

Reply:
it passes the whole object through, instead of just a specified attribute? wasn't sure if there was a better way to go about this...